src/Setup/BuildPlan.purs

```diff
@@ -52,19 +49,24 @@ getVersionField key = do
         pure (pure (Exact version))
 
 -- | Resolve the exact version to provide for a tool in the environment, based
--- | on the action.yml file.
-resolve :: Json -> Tool -> ExceptT Error Effect (Maybe { tool :: Tool, version :: Version })
+-- | on the action.yml file. In the case of NPM packages, bypass the action.yml
+-- | file and use 'Latest'.
```

Comment:
This isn't quite right -- we don't want to bypass the `action.yml` because that's where the user specifies "latest" or a specific version. If they give a specific version then we want to use that. It's rather that we want to bypass the versions file when the user wants the 'latest' version of an NPM-installed package.